### PR TITLE
feat: v1.13.x compatibility

### PR DESCRIPTION
**Pull request by julianlam**
- Changed the way the js is embedded into the page. It is now
  added once on page load, and the widgets are loaded on topic
  and post load.

Hi @dovydaskukalis -- this PR refactors the logic so it works with v1.13.x

If you don't want to maintain this plugin anymore, just grant me (julianlam on npm) publish rights on npm and we can take it over :+1: 

Otherwise merge and publish please :smile:

### CODE REVIEW

File: static/lib/main.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	document.head.appendChild(scriptEl);
+
+	function enhanceEmbed() {
+		console.log('loadin\'');
+		var target = $('ul[component="topic"]').get(0);
+		twttr.widgets.load(target);
+	}
+
+	$(window).on('action:posts.loaded action:topic.loaded', function () {
+		console.log('posts loaded');
```
Comment (barisusakli) [inline static/lib/main.js]: 🎅

File: static/lib/main.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	// Inject twitter widget into footer
+	var scriptEl = document.createElement('script');
+	scriptEl.async = 'async';
+	scriptEl.src = '//platform.twitter.com/widgets.js';
+	scriptEl.charset = 'utf-8';
+
+	document.head.appendChild(scriptEl);
+
+	function enhanceEmbed() {
+		console.log('loadin\'');
```
Comment (barisusakli) [inline static/lib/main.js]: 🤶

**Comment by dovydaskukalis** [comment]
Hi @julianlam . Yea, unfortunately I don't use NodeBB in any of my projects now so I won't be able to maintain it. If you want I can transfer repository to your organisation, just let me know where :)

**Comment by dovydaskukalis** [comment]
Hi @julianlam. Yeah, unfortunately I don't use NodeBB for any of my current projects so I am not being able to maintain it. Invited you to npm.

**Comment by julianlam** [comment]
@dovydaskukalis No problem :smile: We'll take it over under the NodeBB-Community umbrella!

**Comment by julianlam** [comment]
@barisusakli :dog:

**Comment by dovydaskukalis** [comment]
@julianlam Awesome! I will mark this repo as archived and reference your forked repo in readme 🐶

**Pull request closed without merging**